### PR TITLE
small markup changes

### PR DESCRIPTION
**Pull request by Jay-o-Way**

### CODE REVIEW

**Comment by Jay-o-Way** [comment]
#sign-off

**Comment by Jay-o-Way** [comment]
Note: the page https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack&ssr=false#overview has some spelling errors. How to update that page?

**Comment by jehchow** [comment]
> Note: the page https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack&ssr=false#overview has some spelling errors and a broken link (at least the "MIT" one). How to update that page?

Thanks @Jay-o-Way, I'll be reviewing this PR and I'll let you know if I have any requested changes. As far as the Visual Studio marketplace page goes, I'll let the PM of that product know of the typos and the broken link. Thank you for bringing that to our attention!

File: Contribute/how-to-write-docs-auth-pack.md [hunk 1]
```diff
@@ -129,25 +129,25 @@ The content team will review your template and merge the PR if it meets the styl
...
+  - Support for "Learn: Link to file in repo"
+- **Markdown files**
+  - Update "ms.date" Metadata Value context menu option
+  - Code auto-completion support for code-fence language identifiers
+  - Unrecognized code-fence language identifier warnings / auto correction support
+  - Sort selection ascending (A to Z)
+  - Sort selection descending (Z to A)
 
-> [!VIDEO https://www.youtube.com/embed/6zfbBRdjlw8]
+> [!VIDEO <https://www.youtube.com/embed/6zfbBRdjlw8>]
```
Comment (jehchow) [inline Contribute/how-to-write-docs-auth-pack.md:141]:
```suggestion
> [!VIDEO https://www.youtube.com/embed/6zfbBRdjlw8]
```

**Review by jehchow — COMMENTED** [review]
@Jay-o-Way Thank you for your contribution! Please change back the syntax for the embedded video. Also, I've left the markdown edits for italics in, but please note that either a single asterisk OR a single underscore is correct on our platform. 

Once you've changed back the syntax for the embedded video, I will merge this PR. Thank you!

File: Contribute/how-to-write-docs-auth-pack.md (hunk 1 above)
Comment (Jay-o-Way) [reply Contribute/how-to-write-docs-auth-pack.md:141]: Oh that was unintended. Must have been the Learn Author Pack that did it on a general formatting.

**Comment by Jay-o-Way** [comment]
> Also, I've left the markdown edits for italics in, but please note that either a single asterisk OR a single underscore is correct on our platform.

Yeah, you're right. It's something you'll notice when one `*`, indicating a list, is followed by one or two `*` for formatting. That's why I try to follow GitHub's "habit" on pressing Ctrl+B or Ctrl+I or something like that.

**Review by jehchow — APPROVED** [review]

**Pull request merged**